src/renderer/index.css

```
@@ -1,4 +1,6 @@
-@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap');
+/* Local variable fonts (no network dependency, consistent across OS) */
+@import url('@fontsource-variable/inter/index.css');
```

Stylelint (17.12.0)

[error] Expected "url('`@fontsource-variable/inter/index.css`')" to be "'`@fontsource-variable/inter/index.css`'" (import-notation)

(import-notation)